js/components/MilkdropWindow/index.tsx

```diff
@@ -117,8 +118,8 @@ function Milkdrop(props: Props) {
       props.selectNextPreset,
       MILLISECONDS_BETWEEN_PRESET_TRANSITIONS
     );
-    return () => clearImmediate(intervalId);
```

Comment:
Oops!

Reply:
:D

When I initially made the change it was adding an extra timer for each preset change which was...hilarious hahah

Reply:
It would be cool if TypeScript used an opaque type here, or something.